### ISSUE TITLE
Tabs role="presentation" isn't a11y compliant

### ISSUE THREAD

**Issue by BitBrit** [kaliber5/ember-bootstrap#781]
When running a11yAudit(); I am seeing the following issue:

```
Error: Assertion Failed: The page should have no accessibility violations. Violations:
[serious]: <ul> and <ol> must only directly contain <li>, <script> or <template> elements 
Violated 1 time. Offending nodes are: 
<ul id="ember553" class="nav nav-tabs nav-justified ember-view">
```

Here is the "offending" markup.

<img width="597" alt="Captura de pantalla 2019-03-21 a la(s) 3 40 54 p  m" src="https://user-images.githubusercontent.com/1338054/54790355-11d7c400-4bf3-11e9-9e31-23cbf8602a16.png">

I asked Mel about this on Discord and she pointed out that the `role="presentation"` changes the `li` so that it's not `li` anymore.

I looked to see if this role was added by bootstrap itself but it doesn't appear to be:

https://getbootstrap.com/docs/4.3/components/navs/#tabs

Then I looked that the ember-bootstrap tabs demo page:

https://www.ember-bootstrap.com/#/components/tabs

And inspected the tab examples which showed:

<img width="593" alt="Captura de pantalla 2019-03-21 a la(s) 4 08 07 p  m" src="https://user-images.githubusercontent.com/1338054/54790486-8ca0df00-4bf3-11e9-8bfb-3f1e53409db7.png">

So it appears that ember-boostrap adds the `role="presentation"`.

**Comment by simonihmig** [comment]
Thanks @BitBrit for reporting! This is a relict from the BS3 examples: https://getbootstrap.com/docs/3.4/components/#nav

@jelhan was super quick to provide a PR: #782, see my comment there, and feel free to chime in if you have thoughts!
